crates/bevy_winit/src/winit_windows.rs

```diff
@@ -194,7 +194,7 @@ impl WinitWindows {
             use winit::platform::web::WindowExtWebSys;
 
             if window.canvas.is_none() {
-                let canvas = winit_window.canvas();
+                let canvas = winit_window.canvas().unwrap();
```

Comment:
I think it's alright to call unwrap on that, I believe it's instantiated at [the same time of a window](https://github.com/rust-windowing/winit/blob/master/src/platform_impl/web/window.rs#L44), so it might fail if you're doing convoluted things like modifying the canvas from javascript.

But that's not a rabbit hole I'm ready to follow yet. I'll make a note about it.

Reply:
> Only returns the canvas if called from inside the window.

Apparently this didn't turn out very clear. This will only return `None` if called from a Web worker, it will always succeed if called from the "main thread" (the "window" context).